### PR TITLE
332 add claims supported

### PR DESCRIPTION
**Pull request by zacps**

### CODE REVIEW

File: oidc_provider/tests/cases/test_provider_info_endpoint.py [hunk 1]
```diff
@@ -18,12 +24,61 @@ def test_response(self):
...
+            'subject_types_supported',
+            'token_endpoint_auth_methods_supported',
+            'claims_supported',
+        }
+
+        request = self.factory.get(self.url)
+
+        response = ProviderInfoView.as_view()(request)
+        resp_keys = set(json.loads(response.content.decode('utf-8')).keys())
+        self.assertEqual(required_keys, resp_keys)
```
Comment (adamlynam-catalyst) [inline oidc_provider/tests/cases/test_provider_info_endpoint.py]: I think the convention here is `assertEqual(observed, expected)` so these should be reversed?

```suggestion
        self.assertEqual(resp_keys, required_keys)
```

File: oidc_provider/tests/cases/test_provider_info_endpoint.py [hunk 2]
```diff
@@ -18,12 +24,61 @@ def test_response(self):
...
 
         response = ProviderInfoView.as_view()(request)
 
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response['Content-Type'] == 'application/json', True)
         self.assertEqual(bool(response.content), True)
+
+    def test_expected_keys_in_response(self):
+        """
+        Test that response contains all ncecessary keys
```
Comment (adamlynam-catalyst) [inline oidc_provider/tests/cases/test_provider_info_endpoint.py]:
```suggestion
        Test that response contains all necessary keys
```

**Review by adamlynam-catalyst — APPROVED** [review]
LGTM. I think we should make the minor changes I suggested but the overall addition looks great.

**Comment by zacps** [comment]
Great, I'll merge this once we have a pipeline setup.

**Pull request merged**